src/iohumdrum.cpp

```diff
@@ -631,7 +716,7 @@ string HumdrumInput::getDateString(void)
 {
     time_t t = time(0); // get time now
     struct tm *now = localtime(&t);
-    string dateStr = StringFormat("%d-%02d-%02d %02d:%02d:%02d", now->tm_year + 1900, now->tm_mon + 1, now->tm_mday,
+    string dateStr = StringFormat("%d-%02d-%02dT%02d:%02d:%02d", now->tm_year + 1900, now->tm_mon + 1, now->tm_mday,
```

Comment:
Is putting a "T" between the date and time standard?  I think I copied this from elsewhere in the verovio code, so it would be good to have them match one way or the other.